admin_guide/quota.adoc

```diff
@@ -211,13 +234,15 @@ the system.
 When allocating
 xref:../dev_guide/compute_resources.adoc#dev-compute-resources[compute
 resources], each container may specify a request and a limit value each for
-CPU and memory. Quotas can restrict any of these values.
-
-If the quota has a value specified for `*requests.cpu*` or `*requests.memory*`,
-then it requires that every incoming container make an explicit request for
-those resources. If the quota has a value specified for `*limits.cpu*` or
-`*limits.memory*`, then it requires that every incoming container specify an
-explicit limit for those resources.
+CPU, memory, and ephemeral storage. Quotas can restrict any of these values.
+
+If the quota has a value specified for `*requests.cpu*`,
+`*requests.memory*`, or `*ephemeral-storage*`, then it requires that
+every incoming container make an explicit request for those
+resources. If the quota has a value specified for `*limits.cpu*`,
+`*limits.memory*`, or `*ephemeral-storage`, then it requires that
+every incoming container specify an explicit limit for those
```

Comment:
same comment here, ephemeral storage is not required.

Reply:
+1